ethereum/test/test_mmr_verification.js

```diff
@@ -392,4 +415,32 @@ describe("MMRVerification Contract", function () {
             expect(await mmrVerification.verifyInclusionProof.call(root, leafNodeHash, proof.leaf_index, proof.leaf_count, proof.items)).to.be.true
         });
     })
-})
+
+    context("7-leaf MMR from fixture", function () {
```

Comment:
nit: seems like the fixture test cases can now replace the 7-leaf and 15-leaf test cases above? No need to change in this PR though